From: Václav Ovsík <vaclav.ovsik@gmail.com>
Subject: [PATCH] fixes/doc-html-local-css

This patch replaces remote link to CSS with local copy.
Original link: http://www.and.org/f_c-1.0.css

Signed-off-by: Václav Ovsík <vaclav.ovsik@gmail.com>

---
 Documentation/design.html   | 2 +-
 Documentation/f_c-1.0.css   | 1 +
 Documentation/index.html    | 2 +-
 Documentation/tutorial.html | 2 +-
 4 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/Documentation/design.html b/Documentation/design.html
index a13e950..5ccc8ff 100644
--- a/Documentation/design.html
+++ b/Documentation/design.html
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>Design of ustr, the micro string API - for C</title>
-    <link rel="stylesheet" type="text/css" href="http://www.and.org/f_c-1.0.css"
+    <link rel="stylesheet" type="text/css" href="f_c-1.0.css">
 
     <style>
       body { width 85% };
diff --git a/Documentation/f_c-1.0.css b/Documentation/f_c-1.0.css
new file mode 100644
index 0000000..d96ea41
--- /dev/null
+++ b/Documentation/f_c-1.0.css
@@ -0,0 +1 @@
+pre.c2html{color:#111;background:#fff;border-left:0.4em solid #eee}pre.c2html span.comment{color:#292}pre.c2html span.stdout,pre.c2html span.stdin,pre.c2html span.intjmin,pre.c2html span.intjmax,pre.c2html span.intmin,pre.c2html span.intmax,pre.c2html span.charbi,pre.c2html span.compline,pre.c2html span.compfile,pre.c2html span.compversion,pre.c2html span.compdate,pre.c2html span.exit,pre.c2html span.inline,pre.c2html span.restrict,pre.c2html span.const,pre.c2html span.static,pre.c2html span.extern,pre.c2html span.unsigned,pre.c2html span.void,pre.c2html span.uintmaxt,pre.c2html span.intmaxt,pre.c2html span.off64t,pre.c2html span.offt,pre.c2html span.sizet,pre.c2html span.ssizet,pre.c2html span.double,pre.c2html span.float,pre.c2html span.long,pre.c2html span.int,pre.c2html span.short,pre.c2html span.char,pre.c2html span.break,pre.c2html span.return,pre.c2html span.cppinclude,pre.c2html span.cppdefine,pre.c2html span.cppdefined,pre.c2html span.cppendif,pre.c2html span.cppelif,pre.c2html span.cppelse,pre.c2html span.cppifdef,pre.c2html span.cppifndef,pre.c2html span.cppif,pre.c2html span.goto,pre.c2html span.abort{color:#628}pre.c2html span.exitsucs,pre.c2html span.chr,pre.c2html span.str{color:#292;font-weight:bold}pre.c2html span.null,pre.c2html span.false,pre.c2html span.true,pre.c2html span.default,pre.c2html span.case,pre.c2html span.switch,pre.c2html span.for,pre.c2html span.while,pre.c2html span.do,pre.c2html span.else,pre.c2html span.if,pre.c2html span.mpzt,pre.c2html span.pollfd,pre.c2html span.vstrfmt,pre.c2html span.vstrsects,pre.c2html span.vstrbase{color:#44f}pre.c2html span.num0,pre.c2html span.errno,pre.c2html span.assert,pre.c2html span.warn,pre.c2html span.err,pre.c2html span.stderr,pre.c2html span.exitfail{color:#f22}pre.c2html span.numm1{color:#f22;font-weight:bold}
diff --git a/Documentation/index.html b/Documentation/index.html
index ea6fa9d..09af75c 100644
--- a/Documentation/index.html
+++ b/Documentation/index.html
@@ -5,7 +5,7 @@
    <meta http-equiv="Content-type" content="text/html; charset=utf-8">
 
     <title>µstr - Micro String API - for C</title>
-    <link rel="stylesheet" type="text/css" href="http://www.and.org/f_c-1.0.css">
+    <link rel="stylesheet" type="text/css" href="f_c-1.0.css">
 
 
     <style>
diff --git a/Documentation/tutorial.html b/Documentation/tutorial.html
index fe7dd64..a21717d 100644
--- a/Documentation/tutorial.html
+++ b/Documentation/tutorial.html
@@ -3,7 +3,7 @@
   <head>
     <title>Tutorial on ustr APIs</title>
 
- <link rel="stylesheet" type="text/css" href="http://www.and.org/f_c-1.0.css">
+ <link rel="stylesheet" type="text/css" href="f_c-1.0.css">
   <style>
       A       { background: #FFF; color: #44F; }
       A:hover {                   color: #20b2aa; }
-- 
tg: (b148fb7..) fixes/doc-html-local-css (depends on: upstream)
